README.md

```diff
@@ -35,6 +35,17 @@ Featured in the video:
 * Showing diagnostic message in a floating window.
 * Code actions.
 
+## How to get the most out of this plugin
+
+This plugin is designed to provide an opinionated and useable set of defaults for working with LSP and Completion in Neovim. A few things should be considered before you use it in your own config
+
+* To get the best experience, it is recommended to read through this page from top to bottom and to understand the options available to you. 
+* Many things that users often try to customize outside of this plugin are handled through the plugin functionality, and the "doubling up" of configuration can cause issues and loss of functionality. If you are using this plugin and want to customize anything, it is recommended to follow the instructions below and in the [Advance usage](https://github.com/VonHeikemen/lsp-zero.nvim/blob/main/advance-usage.md) page. 
+* Any setup code used outside the built in configuration options and related to LSP or Completion may result in undesired behavior. 
```

Comment:
Think this is too scary. Sounds like lsp-zero should control everything. Maybe there are plugins related to LSP that could be configured outside lsp-zero.

Maybe we could be more specific here? Like list specific situations that can lead to undesired behavior.

Reply:
That language wasn't sitting well with me either TBH. I've been toying with how to rephrase that. What solutions do you think should be listed out?

Reply:
I can't think of anything. Most issues are about overriding some default value or keybinding.

Lets just start with the `lspconfig` thing. Lets say specifically that calling `lspconfig` manually outside of lsp-zero will cause problems.

Reply:
Added.